src/codegen/generators/statement_generator.rs

```diff
@@ -220,6 +224,101 @@ impl<'a, 'b> StatementCodeGenerator<'a, 'b> {
         Ok(())
     }
 
+    fn generate_direct_access_assignment(
+        &self,
+        left_statement: &AstStatement,
+        right_statement: &AstStatement,
+    ) -> Result<(), CompileError> {
+        //TODO : Validation
+        let exp_gen = self.create_expr_generator();
+        if let AstStatement::QualifiedReference { elements, .. } = left_statement {
+            //Target
+            let target: Vec<AstStatement> = elements
```

Comment:
I like this implementation.
Do you have a test that acutally checks the qualified-access? I cant find one - can we add one?

like: a.b.c.%b1 := 100;